src/keymap.cpp

```diff
@@ -35,6 +35,8 @@ static op_desc opdescs[] = {
 	{ OP_RANDOMUNREAD,		"random-unread",			"^K",	_("Go to a random unread article"),		KM_FEEDLIST | KM_ARTICLELIST | KM_ARTICLE },
 	{ OP_OPENBROWSER_AND_MARK, "open-in-browser-and-mark-read",                   "O",    _("Open article in browser and mark read"),           KM_ARTICLELIST },
 	{ OP_OPENINBROWSER,		"open-in-browser",			"o",	_("Open article in browser"),			KM_FEEDLIST | KM_ARTICLELIST | KM_ARTICLE },
+	{ OP_OPENALLINBROWSER, "open-all-in-browser",                   "NOKEY",    _("Open all unread items of selected feed in browser"),          KM_FEEDLIST | KM_ARTICLELIST },
```

Comment:
Instead of `"NOKEY"`, specify an empty string. This does the right thing in terms of listing the operations in the help dialog as "unbound".